dask_searchcv/model_selection.py

```diff
@@ -866,10 +876,12 @@ def visualize(self, filename='mydask', format=None, **kwargs):
     either binary or multiclass, ``StratifiedKFold`` is used. In all
     other cases, ``KFold`` is used.
 
-refit : boolean, default=True
+refit : boolean, array-like or (X, y) tuple, default=True,
     Refit the best estimator with the entire dataset.
     If "False", it is impossible to make predictions using
-    this {name} instance after fitting.
+    this {name} instance after fitting.  If passing
```

Comment:
I think adding a short explanation of what happens when passing an `X` or tuple does here would be helpful.